bundle/manifests/netobserv-metrics-monitor_monitoring.coreos.com_v1_servicemonitor.yaml

```
@@ -13,7 +13,7 @@ spec:
     scheme: https
     tlsConfig:
       caFile: /etc/prometheus/configmaps/serving-certs-ca-bundle/service-ca.crt
-      serverName: netobserv-metrics-service.openshift-operators.svc
+      serverName: netobserv-metrics-service.openshift-netobserv-operator.svc
```

```suggestion
      serverName: netobserv-metrics-service.netobserv-operator.svc
```

Should we use `netobserv-operator` upstream and `openshift-netobserv-operator` downstream ?